package.json

```diff
@@ -25,10 +25,10 @@
     "@types/node": "12.12.7",
     "@types/semver": "6.2.0",
     "eslint": "6.6.0",
-    "jest": "24.9.0",
+    "jest": "25.5.4",
```

Comment:
Other than having to upgrade to `TS >= 3.8` what's preventing us from upgrading to `jest 26.0.1`?

Reply:
well I could not get it to build with the newest version due to typescript issues. I believe that is the only prevention but I have not been able to test running our test script with the newest version

Reply:
Jest 25 is definitely an improvement over 24. I'll take it! 👍 

Perhaps upgrade to TS 3.8 on Engineering Improvement day. 😄